how-to-guides/databases/connecting-to-mongodb.md

```diff
@@ -0,0 +1,476 @@
+{% endtab %}
+{% endtabs %}
+
+5. You will see the mongosh shell if the connection is authenticated.
+
+![MongoDB Shell Access](../../.gitbook/assets/mongodb-shared-access.png)
+
+{% endtab %}
+{% endtabs %}
+
+## New MongoDB Cluster
+
+If you need to make a new database cluster and access it through DevBox, then follow the below steps:
+
+{% tabs %}
+{% tab title="Dedicated or Serverless" %}
+
+### Step 1: Creating a Cluster
+
+1. Go to **[MongoDB Atlas](https://cloud.mongodb.com/v2)**.
+2. Go to **Database > Clusters** and click on **Create**.
+3. Choose between **Dedicated** or **Serverless** database cluster type.
+4. Enter the **Instance name**, **Provider**, and **Region**.
+5. Click on **Create Deployment** and give it time to deploy the infrastructure.
+6. Go to **Security > Network Access** and open the **Private Endpoint** tab.
+7. Click on **Add Private Endpoint** and choose your cloud provider. For this tutorial, we will be using Azure.
+
+**_NOTE:_** You must provide the billing information in the Edit Payment Method form if you don't have a payment method already configured for your organization.
+
+8. Click on **Next** and choose the **Region** where you want to deploy your endpoint.
+9. Enter your **Resource Group Name**, **Virtual Network Name**, **Subnet Name**, and the name you want to give to your endpoint.
+10. Copy and run this PrivateLink Endpoint Command where your Azure CLI is configured. Remember that you can only copy the command once Atlas finishes creating VNET resources in the background for internal use.
+
+![MongoDB AZ CLI](../../.gitbook/assets/mongodb-endpoint-command.png)
+
+11. Copy the **Private Endpoint Resource ID** and **Private Endpoint IP Address** and paste them into the **MongoDB Private Endpoint** Dialog box after creating the private endpoint.
+12. Click on **Create**, and your private endpoint will be created.
+
+![MongoDB endpoint list](../../.gitbook/assets/mongodb-endpoint-list.png)
+
+### Step 2: Installing MongoDB Shell tool in DevBox
+
+To connect with the cluster, we need to install the `mongosh` shell tool.
+
+Follow the below steps to do so:
+
+1. Install `gnupg` and `curl` if they are not already:
+
+{% code %}
+```bash
+sudo apt-get install gnupg curl
+```
+{% endcode %}
+
+2. Get the **MongoDB public GPG key**:
+
+{% code %}
+```bash
+curl -fsSL https://www.mongodb.org/static/pgp/server-7.0.asc | \
+   sudo gpg -o /usr/share/keyrings/mongodb-server-7.0.gpg \
+   --dearmor
+```
+{% endcode %}
+
+3. Create a list file for MongoDB:
+
+{% code %}
+```bash
+echo "deb [ arch=amd64,arm64 signed-by=/usr/share/keyrings/mongodb-server-7.0.gpg ] https://repo.mongodb.org/apt/ubuntu jammy/mongodb-org/7.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-7.0.list
+```
+{% endcode %}
+
+4. Install the MongoDB Package:
+
+{% code %}
+```bash
+sudo apt-get install -y mongodb-org
+```
+{% endcode %}
+
+5. Start the MongoDB Daemon (`mongod`):
+
+{% code %}
+```bash
+sudo systemctl start mongod
```

Comment:
why is mongod needed for client-side?

Reply:
it was a recommended step in the mongosh installation docs, but now that you have pointed it out, it really doesn't serve much purpose. Will have it removed.